src/main/java/com/enonic/app/rewrite/requesttester/RequestTesterImpl.java

```diff
@@ -59,7 +59,7 @@ private RequestTesterResult doExecute( final String host, final String requestPa
     {
         final RequestTesterResult.Builder builder = RequestTesterResult.create();
 
-        final String requestURL = Paths.get( host, requestPath ).toString();
+        final String requestURL = PathBuilder.create( host, requestPath ).build();
```

Comment:
This looks scary. 
"example.com" + /a" gives "example.com/a" - it is meaningless because host is not a part of path in URL

Reply:
it was discussed